### PR TITLE
New Doc Processes Files_Edits for DOC-5

### PR DESCRIPTION
**Pull request by janettelynch**
Add glossary and styleguide. These are a work in progress. I wanted to get them into GitHub so I can link to the files from the wiki.

### CODE REVIEW

**Review by AnneNielsenJ1 — DISMISSED** [review]
Thanks for fixing the line breaks!

File: docs/jupiterone-query-language.md [hunk 1]
```diff
@@ -247,19 +247,24 @@ The above query would return entities of the `Host` class with any of the follow
 
 ## Parameters
 
-The query language supports [parameters](./parameters.md) for referencing values stored on the server-side.  Parameter expressions are allowed in places one might include a literal value otherwise.
+The query language supports [parameters](./parameters.md) for referencing values 
+stored on the server side. Parameter expressions are allowed in places,
```
Comment (oliver-io) [inline docs/jupiterone-query-language.md:251]: I believe the comma after `places` here should be removed.

File: docs/parameters.md [hunk 2]
```diff
@@ -152,12 +156,12 @@ mutation Mutation($name: String!) {
 ```
 <br>
 
-## Referencing Parameters
+## Parameter References
 
-Parameters can be referenced in [rules' configurations](./schemas/alert-rule.md) or any [query expression](./jupiterone-query-language.md), though the syntax is slightly different.  Inside of queries, the dollar-sign-bracket syntax can be used to reference objects; the `param` object is a special member, which when invoked, will fetch from the parameter-storing service to retrieve values.  In the case of both rules and queries, references to parameters which don't exist will cause errors and abandon execution.
+You can reference parameters in [rules configurations](./schemas/alert-rule.md) or any [query expression](./jupiterone-query-language.md), although the syntax is slightly different.  Inside of queries, you can use the dollar-sign-bracket syntax to reference objects. The `param` object is a special member which, when invoked, fetches values from the parameter-storing service.  In the case of both rules and queries, references to parameters which do not exist causes errors and abandons execution.
```
Comment (oliver-io) [inline docs/parameters.md:161]: The change in tenses here convoluted the last sentence, may I suggest: `In the case of both rules and queries, referenced parameters which do not exist will cause errors and abandon execution.`

File: docs/parameters.md [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 
- - Long or unwieldy values (*e.g.* a long URL)
- - Sensitive values (*e.g.* a private key or API token)
- - Common values (*e.g.* dates, keys) we may want to change in many places at once
+ - Long or unwieldy values (such as a long URL)
+ - Sensitive values (such as a private key or API token)
+ - Common values (such as dates, keys) that you may want to change in many places at one time
 
- Today, a better alternative exists: parameters can be stored and referenced in rules and queries with a special syntax.
+ A better alternative exists as parameters that can be stored and referenced in rules and queries with a special syntax.
```
Comment (oliver-io) [inline docs/parameters.md:9]: Possibly change to "A better alternative exists in the form of . . . "?

File: docs/parameters.md (hunk 2 above)
Comment (oliver-io) [inline docs/parameters.md:161]: Yoinking out the semicolon here is probably good, but it creates a sentence fragment with no proper subject.  Can we change to: "`param` is a special keyword in queries which, when invoked, fetches values from the parameter-storing service."

**Review by oliver-io — COMMENTED** [review]
Looks great Janette!  Thank you.

File: docs/jupiterone-query-language.md (hunk 1 above)
Comment (janettelynch) [reply docs/jupiterone-query-language.md:251]: Removed comma and changed which to that.

File: docs/parameters.md (hunk 3 above)
Comment (janettelynch) [reply docs/parameters.md:9]: Agreed. Changed.

File: docs/parameters.md (hunk 2 above)
Comment (janettelynch) [reply docs/parameters.md:161]: Changed.

Comment (janettelynch) [reply docs/parameters.md:161]: We use the present tense to say that a causation or a truth exists now and not something that we will release in the feature. Kinda like saying "Lack of dental care causes tooth decay" vs. "Lack of dental care will cause tooth decay." We have decided to only use the future tense on features/code not yet released.

**Review by AnneNielsenJ1 — APPROVED** [review]
Looks good - thank you Janette!

**Review by oliver-io — APPROVED** [review]

**Pull request merged**